front/app/containers/Admin/reporting/components/ReportBuilder/Analysis/Analyses.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import { Text } from '@citizenlab/cl2-component-library';
+
+import useAnalyses from 'api/analyses/useAnalyses';
+
+import { useIntl } from 'utils/cl-intl';
+
+import Insights from './Insights';
+import messages from './messages';
+
+const Analyses = ({
+  projectId,
+  phaseId,
+  questionId,
+  selectedLocale,
+}: {
+  projectId?: string;
+  phaseId?: string;
+  questionId?: string;
+  selectedLocale: string;
+}) => {
+  const { formatMessage } = useIntl();
+  const { data: analyses, isLoading } = useAnalyses({
+    projectId: phaseId ? undefined : projectId,
+    phaseId,
+  });
+
+  const relevantAnalyses = questionId
+    ? analyses?.data.filter(
+        (analysis) =>
+          analysis.relationships.main_custom_field?.data.id === questionId
+      )
+    : analyses?.data;
+
+  if (relevantAnalyses?.length === 0 && !isLoading) {
+    return <Text>{formatMessage(messages.noInsights)}</Text>;
```

Comment:
Instead of just saying "there are no insights", I think we should actually link to the sensemaking tool where you can create these insights? 

Also, is it possible to check if there are any insights at all for this project? And if not, don't make people go through all the phase and question selection steps, but instead just immediately say "There are no insights, create some [here]"?